independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/ComputingCache.java

```diff
@@ -80,18 +80,24 @@ public void clear() {
 
     public void forEachValue(Consumer<? super V> action) {
         Objects.requireNonNull(action);
-        for (LazyValue<V> value : map.values()) {
-            action.accept(value.get());
-        }
+        map.forEach(new BiConsumer<K, LazyValue<V>>() {
```

Comment:
I'm curious how much better this appoach could be. From what I see in the `ConcurrentHashMap` the `ValuesView` returned from the `values()` is cached and here you create a new `BiConsumer` for every invocation. There is also no lambda here...

Reply:
Yeah, this wasn't meant to remove lambdas, it was meant to take advantage of the better access that `forEach` has for maps. But I'll remove it as to not confuse the scope of the PR